citus-rebalancer.spec

```diff
@@ -47,8 +47,21 @@ PATH=%{pginstdir}/bin:$PATH
 %{pginstdir}/lib/shard_rebalancer.so
 %{pginstdir}/share/extension/shard_rebalancer-*.sql
 %{pginstdir}/share/extension/shard_rebalancer.control
+%ifarch ppc64 ppc64le
+ %else
+ %if %{pgmajorversion} >= 11 && %{pgmajorversion} < 90
+   %if 0%{?rhel} && 0%{?rhel} <= 6
+   %else
+     %{pginstdir}/lib/bitcode/shard_rebalancer*.bc
+     %{pginstdir}/lib/bitcode/shard_rebalancer/*.bc
```

Comment:
@velioglu I manually investigated the outputs and added these paths here. If I were to add another line similar to https://github.com/citusdata/packaging/commit/7589e5907ba67de8899e50f33005b203a509a484
```
      %{pginstdir}/lib/bitcode/shard_rebalancer/*/*.bc
```

there does not exist any file matching this wildcards and the build fails.
Locally building and investigating if the rpm package contains all the `*.bc` files is really hard